src/Symfony/Component/Cache/Adapter/PhpArrayAdapter.php

```diff
@@ -328,7 +328,7 @@ public function commit()
      */
     private function initialize()
     {
-        $this->values = @(include $this->file) ?: array();
+        $this->values = file_exists($this->file) ? (include $this->file ?: array()) : array();
```

Comment:
Shouldn't it be `is_file` instead? `file_exists()` returns `true` for directories.